packages/api/tests/mock.factories.js

```diff
@@ -2397,9 +2402,9 @@ async function animal_batchFactory(
     });
 }
 
-async function animal_identifier_typeFactory(rows = 1) {
+async function animal_identifier_typeFactory(rows = 1, key = faker.lorem.word()) {
   return knex('animal_identifier_type')
-    .insert(Array(rows).fill({ key: faker.lorem.word() }))
+    .insert(Array(rows).fill({ key, id: key === 'OTHER' ? 3 : undefined }))
```

Comment:
Another quirk of us deleting the enum tables in test... I do not intend to use it in tests. It is impossible to test adding `other_use` because it fails the constraint without it. When instantiating the table with a migration the constraint of `other_use` takes the value of the `id` of the 'OTHER' column. So it is impossible to test adding `other_use` without having an existing record for when it was first migrated. 

... I would really like us to stop deleting enum tables in test and also stop adding enum table factories.

Reply:
It is number three here: 
![Screenshot 2024-10-21 at 9 41 12 AM](https://github.com/user-attachments/assets/89fffefd-c30a-4add-881b-3d66c2e0fec5)

Reply:
Now I see what's happening! 

> I would really like us to stop deleting enum tables in test and also stop adding enum table factories.

I'd love for us to prioritize that as a high priority tech debt!